application/visualization/genre_breakdown.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+"""Tool for visulizing the genres of a data series
+see -h for help"""
+
+import argparse
+import pandas as pd
+
+def main():
+    """Generates a graph of the breakdown of genres"""
```

Comment:
I'm guessing this is just a temporary placeholder file for me to fill in?

Reply:
yes, I just wanted to make sure that it worked with pickle files

Reply:
It stores all the pandas metadata about the objects in the pkl so that makes reading them much more seamless

Reply:
Perfect, okay